### PR TITLE
OIE Beta Authenticator Management

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- **What's changed?** Authenticator Management for OIE Beta
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+
+**Possession-based:**
+* Phone (SMS, Voice Call)
+* Email
+* WebAuthn
+
+<!--
+## Get started
+
+Explore the Authenticators Administration API: [![Run in Postman](https://run.pstmn.io/button.svg)](%linkhere%). % How to generate this link: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/227648499/How+to+publish+Postman+collections %
```
Comment (thanh-hanguyen-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:28]: Is this visible? We don't have a postman collection right now.

Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:28]: It's commented out.

Comment (thanh-hanguyen-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:28]: update: not visible

**Comment by thanh-hanguyen-okta** [comment]
This looks good to me!

**Review by IanHakes-okta — APPROVED** [review]
🚢

**Pull request merged**